x-pack/plugins/security_solution/cypress/e2e/detection_alerts/investigate_in_timeline.cy.ts

```diff
@@ -62,9 +68,39 @@ describe('Alerts timeline', () => {
   });
 
   it('Add an empty property to default timeline', () => {
+    // add condition to make sure the field is empty
+    openAddFilterPopover();
+    fillAddFilterForm({ key: 'file.name', operator: 'does not exist' });
     scrollAlertTableColumnIntoView(ALERT_TABLE_FILE_NAME_HEADER);
     addAlertPropertyToTimeline(ALERT_TABLE_FILE_NAME_VALUES, 0);
     openActiveTimeline();
     cy.get(PROVIDER_BADGE).first().should('have.text', 'NOT file.name exists');
   });
 });
+
+describe('Bulk Investigate in Timeline', () => {
+  before(() => {
+    cleanKibana();
+    login();
+    createCustomRuleEnabled(getNewRule());
+    esArchiverLoad('bulk_process');
+  });
+
+  after(() => {
+    esArchiverUnload('bulk_process');
+  });
+
+  beforeEach(() => {
+    visit(ALERTS_URL);
```

Comment:
It would be better in terms of execution time, to close the timeline each time instead of reloading the page and waiting for it.

Reply:
@MadameSheema  You are right. Updated.